### ISSUE TITLE
Community around Invidious

### ISSUE THREAD

**Issue by Perflyst** [omarroth/invidious#305]
I am sure this project will grow well. I don't think the right way is to concentrate on invidio.us, the main instance, but rather on the project and the code itself.
We should spread this project and hope that other "self hosters" bring up invidious instances. In the rooms we also have a RSS bot which informs about new commits and releases.

For this I created a Matrix Channel and bridged it to IRC on freenode. You can also join via XMPP.
Matrix: #invidious:matrix.org (or #invidious:snopyta.org)
On Freenode: #invidious
Via XMPP: `#invidious%irc.freenode.net@irc.snopyta.org`

@omarroth maybe you can register the irc channel on freenode, at the moment nobody owns it.
If you like the idea, maybe you can also add it to the README.md and edit your post on reddit about it.

**Comment by elypter** [comment]
to go one step further, it could become a decentralised network that connects all accounts into one distributed platform like diaspora. thats probably quite some work but would be very robust.

**Comment by elypter** [comment]
that sounds like it would be the standard to go with. i think that would be very nice to have.

**Comment by Perflyst** [comment]
but how should this be suitable for a YouTube frontend? ActivityPub is something different (btw Diaspora doesnt even support it). This issue was opened to bring a community together for chat and issue resolving not to make invidious decentralized

**Comment by elypter** [comment]
just you dont confuse it: my idea is to make user profiles decentralised, not the videos. the biggest obstacle why users are no switching platforms of all sorts is because they already have an account on one site but none on the other. if there should be a community of multiple hosted instances then those instances need users. if nobody uses them, then they are pointless and thus also the community around them which probably wont exist in the first place if there arent even any users.

**Comment by omarroth** [comment]
See #264 for some more discussion on that @elypter . 

The discussion here is with regards to opening an IRC channel, Matrix room, or anything else so folks can get help with setting up more easily, not directly related to the way the site works currently. 

I would very much like to continue that conversation, just in #264.

Links to the above have been added to the README, and I'm currently in the process of registering `#invidious` on Freenode. I'll close this once that is complete, unless there's any other method that should be added.

**Comment by Perflyst** [comment]
`#invidious` is now registered on Freenode.
